frontend/src/components/common/Terminal.tsx

```diff
@@ -278,17 +288,28 @@ export default function Terminal(props: TerminalProps) {
       fitAddonRef.current = new FitAddon();
       xtermRef.current.xterm.loadAddon(fitAddonRef.current);
 
-      const command = getCurrentShellCommand();
-
-      xtermRef.current.xterm.writeln(t('Trying to run "{{command}}"…', { command }) + '\n');
-
       (async function () {
-        execRef.current = await item.exec(
-          container,
-          (items: ArrayBuffer) => onData(xtermRef.current!, items),
-          { command: [command], failCb: () => shellConnectFailed(xtermRef.current!) }
-        );
-
+        if (isAttach) {
+          xtermRef?.current?.xterm.writeln(
+            t('Trying to attach to the container {{ container }}...', { container }) + '\n'
```

Comment:
Please use … instead of ..., for consistency and better a11y.

Reply:
udpated

Reply:
There was one missed `...` but I have fixed it.